src/activity_handlers.cpp

```diff
@@ -2116,6 +2119,31 @@ void activity_handlers::hand_crank_do_turn( player_activity *act, player *p )
 
 }
 
+void activity_handlers::hand_pump_integral_do_turn( player_activity *act, player *p )
+{
+    int air_increase = 50;
+    item &hand_pump_integral_item = p->i_at( act->position );
+    //tool section
+    if( hand_pump_integral_item.is_tool() ) {
+        act->moves_left -= 100;
+        item &hand_crank_item_air = p->i_at( act->position );
```

Comment:
Do we need this variable if it's absolutely the same as `hand_pump_integral_item` up above?